docs/ref/models/expressions.txt

```diff
@@ -893,3 +893,53 @@ file (or package) that is imported from the top level ``__init__.py``.
 
 For user projects wishing to patch the backend that they're using, this code
 should live in an :meth:`AppConfig.ready()<django.apps.AppConfig.ready>` method.
+
+Using operators to combine expressions
+--------------------------------------
+
+.. versionadded:: 2.0
+
+.. currentmodule:: django.db.models.expressions
```

Comment:
Should `CombinedExpression` have a convenience import in `django.db.models` like the other public expressions?